packages/block-editor/src/components/block-compare/index.js

```diff
@@ -11,16 +11,21 @@ import { diffChars } from 'diff/lib/diff/character';
  * WordPress dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { Component } from '@wordpress/element';
 import { getSaveContent, getSaveElement } from '@wordpress/blocks';
 
 /**
  * Internal dependencies
  */
 import BlockView from './block-view';
 
-class BlockCompare extends Component {
-	getDifference( originalContent, newContent ) {
+function BlockCompare( {
```

Comment:
Nitpick: you could do `export default function` here:
```suggestion
export default function BlockCompare( {
```